### PR TITLE
MM-50713: add airflow task for extracting github contributors

### PR DESCRIPTION
**Pull request by ifoukarakis**
#### Summary

- [x] Add task that extracts contributors from github. 

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-50713

### CODE REVIEW

File: dags/extract/contributors.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# Create the DAG
+with DAG(
+        "contributors_nightly",
+        default_args=default_args,
+        schedule_interval="0 7 * * *",
+        catchup=False,
+        max_active_runs=1,  # Don't allow multiple concurrent dag executions
+        doc_md=doc_md,
+    ) as dag:
+    extract_contributors = KubernetesPodOperator(
```
Comment (catalintomai) [inline dags/extract/contributors.py:51]: why not `extract_github_contributors` ?

File: dags/kube_secrets.py [hunk 2]
```diff
@@ -62,3 +62,4 @@
 CLEARBIT_KEY = Secret("env", "CLEARBIT_KEY", "airflow", "CLEARBIT_KEY")
 
 GITHUB_TOKEN = Secret("env", "GITHUB_TOKEN", "airflow", "GITHUB_TOKEN")
```
Comment (munish7771) [inline dags/kube_secrets.py:64]: can we remove the existing GITHUB_TOKEN and the task using it ?

**Review by munish7771 — APPROVED** [review]

File: dags/kube_secrets.py (hunk 2 above)
Comment (ifoukarakis) [reply dags/kube_secrets.py:64]: Let's do cleanup as soon as we make sure this one is working.

File: dags/extract/contributors.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    "depends_on_past": False,
+    "owner": "airflow",
+    "on_failure_callback": send_alert,
+    "retries": 0,
+    "retry_delay": timedelta(minutes=1),
+    "sla": timedelta(hours=8),
+    "start_date": datetime(2019, 1, 1, 0, 0, 0),
+}
+
+doc_md = """
```
[the remark this answers is not in the document]
Comment (ifoukarakis) [reply dags/extract/contributors.py:28]: Agree, but let's do it in separate PRs.

**Pull request merged**